src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""
+==============
+dswx_s1_pge.py
+==============
+Module defining the implementation for the Co-registered Single Look Complex (DSWX)
+from Sentinel-1 A/B (S1) PGE.
+"""
+
+from opera.pge.base.base_pge import PgeExecutor
+from opera.pge.base.base_pge import PostProcessorMixin
+from opera.pge.base.base_pge import PreProcessorMixin
+
+
+class DSWxS1PreProcessorMixin(PreProcessorMixin):
+    """
+    Mixin class responsible for handling all pre-processing steps for the DSWX-S1
```

Comment:
Very small nit, but lets be consistent with the other PGE and use "DSWx" (lower case x) where we can